app/src/main/res/layout/list_item_restaurant.xml

```diff
@@ -1,11 +1,65 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:orientation="horizontal">
+    android:layout_height="216dp"
```

Comment:
A lot of the items here and below have a specific height/width value.

1. for different phones these values could cause the screen to look odd. it might be easier to use relative values?
2. dimens file?